packages/edit-site/src/components/navigation-sidebar/navigation-panel/index.js

```diff
@@ -28,7 +28,6 @@ import {
  * Internal dependencies
  */
 import { useLink } from '../../routes/link';
-import MainDashboardButton from '../../main-dashboard-button';
```

Comment:
Should we do the same change to edit-post in order to be consistent?
e.g.: remove packages/edit-post/src/components/header/main-dashboard-button/index.js and its docs
docs/reference-guides/slotfills/main-dashboard-button.md?